docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -173,783 +194,134 @@ To add an instance type, do the following:
 
-    Ensure that the `yugabyte` user has permissions to SSH into the YugabyteDB nodes (as defined in `/etc/ssh/sshd_config`).
+    - `--ask_password` - this flag instructs the script to prompt for a password (which is required if the sudo user requires password authentication).
+    - `--ip` - enter the IP address of the node.
+    - `--mount_points` - enter the mount point configured for the node (typically `/data`). If you have multiple drives, add these as a comma-separated list, such as, for example, `/mnt/d0,/mnt/d1`.
+    - `--install_node_agent` - this flag instructs the script to install the node agent, which is required for YugabyteDB Anywhere to communicate with the instance.
+    - `--api_token` - enter your API token; you can create an API token by navigating to your **User Profile** and clicking **Generate Key**.
+    - `--yba_url` - enter the URL of the machine where you are running YugabyteDB Anywhere, with port 9000. For example, `http://ybahost.company.com:9000`. The node must be able to communicate with YugabyteDB Anywhere at this address.
+    - `--node_name` - enter a name for the node.
+    - `--instance_type` - enter the name of the [instance type](#add-instance-types) to use for the node. The name must match the name of an existing instance type.
+    - `--zone_name` - enter a zone name for the node.
 
-1. If the node is running SELinux and the home directory is not the default, set the correct SELinux ssh context, as follows:
+    For example:
 
     ```bash
-    chcon -R -t ssh_home_t <yugabyte_home>
-    ```
-
-1. Copy the SSH public key to each DB node. This public key should correspond to the private key entered into the YugabyteDB Anywhere provider.
-
-1. Run the following commands as the `yugabyte` user, after copying the SSH public key file to the user home directory:
-
-    ```sh
-    cd ~yugabyte
-    mkdir .ssh
-    chmod 700 .ssh
-    cat <pubkey file> >> .ssh/authorized_keys
-    chmod 400 .ssh/authorized_keys
-    exit   # (exit from the yugabyte user back to previous user)
-    ```
-
-1. Add the following lines to the `/etc/security/limits.conf` file (sudo is required):
-
-    ```text
-    *                -       core            unlimited
-    *                -       data            unlimited
-    *                -       fsize           unlimited
-    *                -       sigpending      119934
-    *                -       memlock         64
-    *                -       rss             unlimited
-    *                -       nofile          1048576
-    *                -       msgqueue        819200
-    *                -       stack           8192
-    *                -       cpu             unlimited
-    *                -       nproc           12000
-    *                -       locks           unlimited
+    /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py \
```

Comment:
minor - would be nice to follow the same order of the arguments in the example as that in the description.